rego/rego_wasmtarget_test.go

```diff
@@ -338,3 +340,53 @@ func TestEvalWasmWithInterQueryCache(t *testing.T) {
 		t.Fatal("Expected server to be called only once")
 	}
 }
+
+func TestEvalWasmWithHTTPAllowNet(t *testing.T) {
```

Comment:
Would it make sense to run the same test with `Target("rego")`, too? 🤔

Reply:
This is a simplified version of the test in http_test.go, which is implicitly run under the rego target. But It'd be easy to do as you suggest also.

Reply:
Happy to leave this up to you -- no need to overdo it. Your call 😃

Reply:
Leaving this as-is, as this is covered elsewhere.